addons/pos_loyalty/static/src/js/Loyalty.js

```diff
@@ -860,6 +863,9 @@ patch(Order.prototype, "pos_loyalty.Order", {
         if (program.is_nominative && !this.get_partner()) {
             return false;
         }
+        if (program.date_from && program.date_from >= new Date()) {
+            return false;
+        }
```

Comment:
Maybe it would make sense to add help only for the last day, because for the start date it is expected that the program is valid on start date.

Reply:
We should never expect the users to know what's expected and what's not, because that's a cultural thing. 1st floor in Japan is the ground floor, and that's one of the best example I got. :wink: